test/README.md

```diff
@@ -0,0 +1,84 @@
+Candid test data
+================
+
+This directory contains Candid test data, to be used to check Candid
+implementations for compliance.
+
+This test suite covers
+
+ * parsing Candid service descriptions
+ * given a Candid type, decoding binary Candid data, asserting success or failure
+ * given a Candid type, parsing textual Candid data, asserting success or failure
+ * comparing such decoded/parsed values, using the Host language’s notion of equality.
+
+Because this test data necessarily has to be host-language agnostic, it is not
+possible to
+
+ * assert the decoded value, as it would be host-language specific
```

Comment:
This bullet point is no longer accurate with the new assertions.

Reply:
Rewritten, thanks.